packages/signature-controller/src/SignatureController.ts

```diff
@@ -413,6 +413,7 @@ export class SignatureController extends BaseControllerV2<
     const messageParamsWithId = {
       ...messageParams,
       metamaskId: messageId,
+      ...(version && { version }),
```

Comment:
As we pass this object to `signMessage` also, we're passing `version` to it twice in different ways.

Would it be cleaner to just pass the version to `requestApproval` as an additional argument  and then add it to the `requestData` in that method? Or alternatively remove the argument from `signMessage` so it can read it from the message params?

Reply:
No harm for sure, but not in all contexts of signature types that's why I didn't directly extend it. But I think an additional parameter to `requestApproval` makes more explicit here, I will go that way and put a small comment.

Reply:
Apparently removing argument is better option here, no need another type also removing couple more lines. Fixed